### DISCUSSION TITLE
Add support for Expo V.41 and up via the usage of expo config plugins

### DISCUSSION THREAD

**Discussion by joseDaKing** [invertase/react-native-firebase#5526] [Ideas]
Add support for Expo V.41 and up via the usage of expo config plugins

**Comment by mikehardy** [comment]
Search project prior to requesting things (exercise for @joseDaKing: post back what you find)

**Reply by joseDaKing** [reply]
Here is a link to the expo config plugin for [react-native-firebase](https://www.npmjs.com/package/with-rn-firebase) and her is a link to the article about the [config plugin](https://bartlomiej-klocek.medium.com/how-to-integrate-react-native-firebase-into-expo-d34712eaf64d). I think you should update the documentation and mention that react-native-firbase can be used with Expo v.42 and up.

**Reply by mikehardy** [reply]
@joseDaKing You did not search in here? Why not? You might like: 

- https://github.com/invertase/react-native-firebase/pulls?q=expo

which shows:

https://github.com/invertase/react-native-firebase/pull/5480
